<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectItemWdt</class>
 <widget class="QWidget" name="ProjectItemWdt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>105</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="content" native="true">
     <property name="styleSheet">
      <string notr="true">#content {
background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: gray;
border-width: 1px;
border-style: outset;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="logoLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>logo</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nameLbl">
        <property name="text">
         <string>Udimi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stateLbl">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0);
font-size: 18px;</string>
        </property>
        <property name="text">
         <string>Active</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="desc1Wdt" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="desc11Lbl">
           <property name="text">
            <string>time this week</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desc12Lbl">
           <property name="text">
            <string>this month</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desc13Lbl">
           <property name="text">
            <string>total</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="desc2Wdt" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="desc21Lbl">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desc22Lbl">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desc23Lbl">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
